app/transformers/rural-payments-portal/business.js

```diff
@@ -0,0 +1,50 @@
+export default (data) => {
```

Comment:
Can this be a named export with a name something like `transformOrganisationToBusiness`, then we can tell from the name what we're putting in and expecting to get out.

☹️ That we can't use TypeScript here because this is so much easier when we have a type for the data

Reply:
sure, on it...